eclair-core/src/main/scala/fr/acinq/eclair/io/PeerConnection.scala

```diff
@@ -460,7 +462,19 @@ object PeerConnection {
   val MAX_FUNDING_TX_ALREADY_SPENT = 10
   // @formatter:on
 
-  def props(nodeParams: NodeParams, switchboard: ActorRef, router: ActorRef): Props = Props(new PeerConnection(nodeParams, switchboard, router))
+  def props(keyPair: KeyPair, conf: PeerConnectionConf, switchboard: ActorRef, router: ActorRef): Props = Props(new PeerConnection(keyPair, conf, switchboard, router))
+
+  case class PeerConnectionConf(chainHash: ByteVector32,
+                                authTimeout: FiniteDuration,
+                                initTimeout: FiniteDuration,
+                                pingInterval: FiniteDuration,
+                                pingTimeout: FiniteDuration,
+                                pingDisconnect: Boolean,
+                                features: ByteVector,
+                                overrideFeatures: Map[PublicKey, ByteVector],
+                                syncWhitelist: Set[PublicKey],
+                                maxRebroadcastDelay: FiniteDuration
+                               )
```

Comment:
nit: indentation

Reply:
it is correctly indented?

Reply:
Well we usually don't put the parenthesis on a new line, there are several instances of this in the codebase: [RouterConf](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala#L74), [RouteRequest](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala#L153), [SendPaymentConfig](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/scala/fr/acinq/eclair/payment/send/PaymentInitiator.scala#L309), and many others. But to be fair there are also a minority of case classes that are ~~wrongly~~ differently indented.

Reply:
Good point, consistency FTW! Fixed with 884efe7e9.